### PR TITLE
Fix 500 error when converting to local dash

### PR DESCRIPTION
**Pull request by parente**
* Simplify jinja logic
* Comment
* Add report layout example
* Handle missing layout metadata cases more gracefully

### CODE REVIEW

File: dashboards_bundlers/local_deploy/jinja_templates/index.html.tpl [hunk 1]
```diff
@@ -70,28 +67,26 @@
...
-                    {%- set layout = none -%}
-                {%- endif -%}
-            {%- endif -%}
+            
             {# Thebe doesn't wrap HTML-converted cells with same structure as seen in Notebook,
                resulting in different styling. Adding necessary classes to match styles. #}
             {%- set textClass = "rendered_html" if (cell.cell_type is defined) and (cell.cell_type == "markdown") else "" -%}
             <div {% if (dashboardLayout == "report") -%}
-                    class="report-cell {{textClass}} {%- if (hidden) -%} dashboard-hidden{%- endif -%}"
+                    class="report-cell {{textClass}} {%- if (hidden) -%} dashboard-hidden {%- endif -%}"
```
Comment (dalogsdon) [inline dashboards_bundlers/local_deploy/jinja_templates/index.html.tpl]: image
We've got a collapsed space between classes here causing hidden cells to be visible. It looks like this was an existing problem.

**Comment by dalogsdon** [comment]
I tested:
1. No dashboard metadata
2. Current report layout
3. Current grid layout
4. Old grid layout (without `layout: "grid"`)
5. **Report layout without `layout: "report"` causes a blank page since it defaults to grid, but there is no cell metadata**
6. Existing layout but with a cell that has no dashboard metadata

These cases work fine aside from the class space problem I noted in the changes. Case 5 above is an interesting one. Should we address that (complicated) or leave it (as it is probably a result of mucking about with the notebook file)?

**Comment by parente** [comment]
I think leave it. There's no backward compatibility issue because report mode didn't exist in notebooks until now.

File: dashboards_bundlers/local_deploy/jinja_templates/index.html.tpl (hunk 1 above)
Comment (parente) [inline dashboards_bundlers/local_deploy/jinja_templates/index.html.tpl]: Good catch. I'll update and push.

**Comment by parente** [comment]
Fixed the css class whitespace problem. Should be all set.

**Pull request merged**